packages/story-editor/src/components/mediaRecording/provider.js

```diff
@@ -81,6 +83,9 @@ function MediaRecordingProvider({ children }) {
   const [audioInput, setAudioInput] = useState(
     localStore.getItemByKey(LOCAL_STORAGE_PREFIX.MEDIA_RECORDING_AUDIO_INPUT)
   );
+  const [videoEffect, setVideoEffect] = useState(
+    localStore.getItemByKey(LOCAL_STORAGE_PREFIX.MEDIA_RECORDING_VIDEO_EFFECT)
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:**
Would it make sense to lazy init the state for performance reasons? Not sure how much of a difference it makes.

Reply:
We use this value as soon as you open the media recording screen not only in settings dialog.